ansible/hoodi.yml

```diff
@@ -51,6 +51,13 @@
       with_items: '{{ nodes_layout[inventory_hostname] }}'
       when: validator_client_service_enabled
       loop_control: { loop_var: node, index_var: idx }
+    - include_role:
+        name: infra-role-rpc-snooper
+        apply:
+          tags: always
+      tags: [ rpc-snooper ]
+      with_items: '{{ nodes_layout[inventory_hostname] }}'
+      loop_control: { loop_var: node, index_var: idx }
```

Comment:
I get that you put this here because it can't go into individual geth/neth/nec sections, but we should run it before BN since if it will be using it it needs it ready before it starts.